docs/source/changelog.rst

```diff
@@ -6,6 +6,20 @@ Changes in KubeSpawner
 
 .. _changelog_09:
 
+KubeSpawner 0.10
+================
+
+0.10 is a small release, with minor changes and fixes.
+
+- Stop pinning an exact kubernetes client version;
+  instead, require kubernetes client >= 7.
+  If desired, pinning should be done in images/installations
+- Expand username template variables in extra_containers
+- Set pod restart policy to OnFailure, so that notebook servers that crash restart automatically
```

Comment:
I assume this is #251 ? actually crashed notebook servers did restart, the fix is for notebook-initiated cull to correctly stop the server, not triggering restart.

Reply:
Thanks for the correction!